recipes.txt

```diff
@@ -1,50 +0,0 @@
-1; Tomato Basil Pasta; 8 oz pasta, 2 cups cherry tomatoes, 2 cloves garlic, 0.25 cup basil, 2 tbsp olive oil; Cook pasta, sauté garlic and tomatoes in oil, add basil, season, mix with pasta.
-2; Chicken Alfredo; 2 boneless chicken breasts, 8 oz fettuccine, 1 cup heavy cream, 1/2 cup grated Parmesan, 2 tbsp butter, 1 pinch salt; Cook chicken, cook pasta, melt butter, add cream, Parmesan, 1 pinch salt, and pepper, toss with pasta and sliced chicken.
```

Comment:
This file can be csv too, just to be consistent.
Obviously this is decent now.

Reply:
This, (but all ) txt files are just  backup files, if the chatgpt mess something. I will only use csv files.